### DISCUSSION TITLE
[Issue] Game crash when loading save Expedition 7

### DISCUSSION THREAD

**Discussion by K-Boom79** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#26] [General]
After starting Expedition 7, and saving the game, whenever I go to resume play and load the save, the game instantly crashes
This starts happening and is easily reproduced after dying the first time. Looks like something gets corrupted.

![crash](https://github.com/user-attachments/assets/3b92279d-8dfd-4503-a3c6-8d30a7669442)

[NMS_crash_1726020372_128776_0x1137D_ST76561197995743166.dmp](https://github.com/user-attachments/files/16956973/NMS_crash_1726020372_128776_0x1137D_ST76561197995743166.dmp)

**Comment by cwmonkey** [comment]
lol Leviathan is absolutely cursed. Checking now.

**Reply by cwmonkey** [reply]
@K-Boom79 Confirmed. I'll see if I can find a save file editor fix. Otherwise I guess the options are:

1. Don't die (you have to find a hermetic seal at a trade station to fix your ship)
2. Beat the expedition without closing the game after you die once 😬

**Reply by cwmonkey** [reply]
@K-Boom79 I wasn't able to recover the save. I thought someone was able to do it before but I can't find the conversation.

I have patched the expedition json file to not reset the save on death and reduce the difficulty settings so you don't get caught in a death loop. You will need to create a new save (unless you can figure out how to un-brick your broken saves)

You can download the new json here: https://cwmonkey.github.io/nms-expeditions/

Also attached to this message.

Patch for this repo coming soon.

[SEASON_DATA_CACHE.JSON](https://github.com/user-attachments/files/16961791/SEASON_DATA_CACHE.JSON)

**Reply by cwmonkey** [reply]
@leodium json updated in 15aa547

Not an ideal fix but there's not much we can do about HG breaking game mechanics.

**Reply by K-Boom79** [reply]
> @leodium json updated in [15aa547](https://github.com/leodium/leodium-ReLive-NoMansSky-Expeditions-Offline/commit/15aa547410c2c56679c61942dc07a12074268227)
> 
> Not an ideal fix but there's not much we can do about HG breaking game mechanics.

Yeah, that workaround seems to make the expedition work fine for me, and I finished it.
I only died once anyway for the milestone and Hermetic Seal plans since I couldn't find any lol. So the mechanic of the expedition wasn't really missed for me. Thank you

**Reply by eSonOfAnder** [reply]
> @K-Boom79 I wasn't able to recover the save. I thought someone was able to do it before but I can't find the conversation.
> 
> I have patched the expedition json file to not reset the save on death and reduce the difficulty settings so you don't get caught in a death loop. You will need to create a new save (unless you can figure out how to un-brick your broken saves)
> 
> You can download the new json here: https://cwmonkey.github.io/nms-expeditions/
> 
> Also attached to this message.
> 
> Patch for this repo coming soon.
> 
> [SEASON_DATA_CACHE.JSON](https://github.com/user-attachments/files/16961791/SEASON_DATA_CACHE.JSON)

Any ideas/suggestions on unbricking a bricked save? The Warning.txt implied that this issue had already been patched, so I was not expecting to run into this issue. I've got a backup of the save, but it's from several days ago. >_< 

Edit: I went ahead and copied over my backup to the main save area, and after enabling network for the game again it pulled down almost the most current save data from the Xbox cloud save, and thankfully just acted like I had finished Expedition 7 w/o finishing any of the major milestones (i.e. where I had left off). Other than some strangeness with my character appearance that I had to re-do, looks like things are back to normal. I'll have a go at Expedition 7 again some other time now that I know I've got the updated JSON to avoid this issue in the future.

**Reply by cwmonkey** [reply]
> The Warning.txt implied that this issue had already been patched

@eSonOfAnder Yeah it was, then they fixed it so we changed it back then they broke it again, heh. This expedition in particular has been plagued with issues.

So did you manually make a backup of your saves before starting the expedition?

**Comment by leodium** [comment]
Argh.. It's gonna be super hot!!! I already manage to die in normal expeditions because I'm so inattentive :(

**Comment by leodium** [comment]
Ok ... On va mettre une explication sous le fichier en + =)
_____________________________
.: Boris Verrechia :.

✉ www.leodium.net
✉ rue de Brivioulle, 18 - 4260 Braives
▲ 0473/82.86.56


.: Leodium.est.Fauto/Graphe :.

Photo/Graph(i)er l'Ombre et la Lumière
Projets exposés : *#NoOneInMyWorld **#SomeOneInMyWorld **#Sensu.Elle&Lui*

www.instagram.com/leodium.est.fautographe





Le mer. 11 sept. 2024 à 12:16, Gerald Burns ***@***.***> a
écrit :
